### DISCUSSION TITLE
How to change my avatar?

### DISCUSSION THREAD

**Discussion by Gaojianli** [dani-garcia/vaultwarden#3299] [Q&A]
![image](https://user-images.githubusercontent.com/19235246/222638779-2a345bc3-ac24-4dea-a6cf-512f6732d08b.png)
<img width="492" alt="image" src="https://user-images.githubusercontent.com/19235246/222639431-81cd834f-7742-4fa2-a9c4-581f6af1e1a7.png">

Hello everyone, i am new to this project and i wonder how to change my avatar? Is vaultwarden support Gravatars?

**Comment by tessus** [comment]
No, gravatar support was removed a while back due to privacy concerns (I think).

**Comment by BlackDex** [comment]
See https://github.com/bitwarden/clients/pull/3600
